src/plugins/discover/public/application/context/services/context_state.ts

```diff
@@ -173,11 +185,33 @@ export function getState({
     },
   ]);
 
+  let stopSyncingFilters = () => {};
```

Comment:
I guess it would be better to use `useRef` for persisting and updating this handler.

Reply:
I tried to change this but realized since this isn't a component or hook I can't call `useRef` from here.